locksmithctl/daemon.go

```diff
@@ -147,6 +197,7 @@ func (r rebooter) lockAndReboot(lck *lock.Lock) {
 
 			continue
 		}
+		getPeriodicAndSleep()
```

Comment:
since this method returns now an error, we can properly handle it:
```suggestion
		if err := getPeriodicAndSleep(); err != nil {
		    dlog.Fatalf("unable to check reboot window period: %v", err)
		}
```

--

we can see the advantages of returning errors from the `getPeriodicAndSleep` function - we now handle the error, we can decide to skip it, log it, abort the program on the error, etc.